### ISSUE TITLE
[Accessibility: Alternative content] User activity Log modal-dialog does not have the required label

### ISSUE THREAD

**Issue by rajatgupta21** [kimchi-project/wok#57]
The div element with a WAI-ARIA role document does not have the required label specified by aria-label or aria-labelledby.
